t/plugin/clickhouse-logger.t

```diff
@@ -214,7 +214,38 @@ passed
 
 
 
-=== TEST 6: access local server
```

Comment:
We prefer to add new test at the bottom to avoid affecting existing tests.

Reply:
Done